### PR TITLE
[Navigation] Remove integrations from new NavGroups

### PR DESCRIPTION
**Pull request by TackAdam**
### Description
Remove integrations from new NavGroups (security-analytics and analytics) and switch to new naming convention Visualize and Report
Screenshot 2024-07-21 at 3 47 24 PM
Screenshot 2024-07-22 at 2 40 45 PM

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/plugin.tsx [hunk 1]
```diff
@@ -351,6 +351,13 @@ export class ObservabilityPlugin
         order: observabilityLogsPluginOrder,
         mount: appMountWithStartPage('logs'),
       });
+      core.application.register({
+        id: observabilityIntegrationsID,
+        title: observabilityIntegrationsTitle,
+        category: DEFAULT_APP_CATEGORIES.management,
+        order: observabilityIntegrationsPluginOrder,
+        mount: appMountWithStartPage('integrations'),
+      });
```
Comment (derek-ho) [inline public/plugin.tsx:360]: Why do we need to re-order this?

File: public/plugin_nav.tsx [hunk 2]
```diff
@@ -33,47 +32,25 @@ export function registerAllPluginNavGroups(core: CoreSetup<AppPluginStartDepende
...
   core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.analytics, [
     {
       id: observabilityNotebookID,
-      category: DEFAULT_APP_CATEGORIES.dashboardAndReport,
+      category: DEFAULT_APP_CATEGORIES.visualizeAndReport,
       order: 400,
     },
   ]);
 
-  core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.observability, [
```
Comment (derek-ho) [inline public/plugin_nav.tsx:55]: Where is integrations moving to?

**Review by derek-ho — APPROVED** [review]

**Review by ps48 — APPROVED** [review]

File: public/plugin_nav.tsx (hunk 2 above)
Comment (TackAdam) [reply public/plugin_nav.tsx:55]: Just saying in observability now removed from security and analytics

**Comment by ps48** [comment]
The snapshot failures are coming in from core.

**Pull request merged**